pom.xml

```diff
@@ -1,90 +1,56 @@
-            <version>${project.version}</version>
-            <type>war</type>
-        </dependency>
-
-        <dependency>
-            <groupId>${geoserver.group.id}</groupId>
-            <artifactId>gs-xslt</artifactId>
-            <version>${project.version}</version>
-            <scope>runtime</scope>
-        </dependency>
-
-        <dependency>
-            <groupId>${geoserver.group.id}.web</groupId>
-            <artifactId>gs-web-wfs</artifactId>
-            <version>${project.version}</version>
-        </dependency>
-
-    </dependencies>
-
-    <build>
-
-        <resources>
-            <resource>
-                <directory>${basedir}/src/main/java</directory>
-                <includes>
-                    <include>applicationContextOverrides.xml</include>
-                </includes>
-            </resource>
-        </resources>
-
-        <plugins>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-war-plugin</artifactId>
-                <version>2.4</version>
-                <configuration>
-                    <overlays>
-                        <overlay>
-                            <groupId>${geoserver.group.id}.web</groupId>
-                            <artifactId>${geoserver.artifact.id}</artifactId>
-                        </overlay>
-                    </overlays>
-                    <warName>geoserver-${project.version}-imos</warName>
-                </configuration>
-            </plugin>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-compiler-plugin</artifactId>
-                <version>3.1</version>
-                <configuration>
-                     <source>1.5</source>
-                     <target>1.5</target>
-                </configuration>
-            </plugin>
-        </plugins>
-    </build>
-
-    <distributionManagement>
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>org.geoserver.web</groupId>
+                <artifactId>gs-web-app</artifactId>
+                <version>${geoserver.version}</version>
+                <type>war</type>
+            </dependency>
+            <dependency>
+                <groupId>org.geoserver</groupId>
+                <artifactId>gs-xslt</artifactId>
+                <version>${geoserver.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.geoserver.web</groupId>
+                <artifactId>gs-web-wfs</artifactId>
+                <version>${geoserver.version}</version>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+
+     <!-- gs-xslt plugin -->
+    <repositories>
         <repository>
-            <id>${internal.repository.id}</id>
-            <url>${internal.repository.url}</url>
+            <id>opengeo</id>
+            <url>http://repo.boundlessgeo.com/main</url>
```

Comment:
I would like to see this for maven projects. But feel this is probably best addresssed in a retro.